Directory.build.props

```diff
@@ -9,31 +9,42 @@
     <PackageTags>mvvm;mvvmcross;xamarin;android;ios;mac;forms;monodroid;monotouch;xamarin.android;xamarin.ios;xamarin.forms;xamarin.mac;wpf;net;net45;uwp;tizen</PackageTags>
     <PackageReleaseNotes>https://github.com/MvvmCross/MvvmCross/blob/develop/CHANGELOG.md</PackageReleaseNotes>
     <PackageRequireLicenseAcceptance>false</PackageRequireLicenseAcceptance>
-    <PublishRepositoryUrl>true</PublishRepositoryUrl>
+    
     <RepositoryUrl>https://github.com/MvvmCross/MvvmCross</RepositoryUrl>
     <RepositoryType>git</RepositoryType>
     <Product>$(AssemblyName) ($(TargetFramework))</Product>
     <NeutralLanguage>en</NeutralLanguage>
-    <Version>6.3.0</Version>
-    
-    <LangVersion>latest</LangVersion>
+    <Version>6.4.0</Version>
+
+    <LangVersion>7.3</LangVersion>

```

Comment:
Latest bundled mono with vs4mac has 8.0 RC support, do we really need to specify 7.3 here?

Reply:
Yeah i don't think it is a good idea to run on RC at the moment. Also a lot of people are not running the latest yet. I think it is a good idea to manually control the number and update it when we are ready for it in the future.